### PR TITLE
Branch/navigation and modal

### PR DESCRIPTION
**Pull request by AbigailGalzinB**
Structure to section navigation:

Screenshot 2024-06-08 174936

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,2 +1,4 @@
 <router-outlet></router-outlet>
-<app-main-content></app-main-content>
+<div class="main">
```
Comment (JuanPabloTraining) [inline src/app/app.component.html]: Why not using a `main` html component directly?

Comment (AbigailGalzin-Dev) [reply src/app/app.component.html]: to avoid the pre-defined styles in angular

Comment (JuanPabloTraining) [reply src/app/app.component.html]: Angular adds predefined styles? O.o

**Review by CatherineGomez-AD — APPROVED** [review]
good job

**Review by JuanPabloTraining — APPROVED** [review]
Acceptedn't

**Pull request merged**